aiohttp/web_fileresponse.py

```diff
@@ -177,13 +180,23 @@ def _get_file_path_stat_encoding(
         return file_path, file_path.stat(), None
 
     async def prepare(self, request: "BaseRequest") -> Optional[AbstractStreamWriter]:
-        loop = asyncio.get_event_loop()
         # Encoding comparisons should be case-insensitive
         # https://www.rfc-editor.org/rfc/rfc9110#section-8.4.1
         accept_encoding = request.headers.get(hdrs.ACCEPT_ENCODING, "").lower()
-        file_path, st, file_encoding = await loop.run_in_executor(
-            None, self._get_file_path_stat_encoding, accept_encoding
-        )
+
+        loop = asyncio.get_running_loop()
```

Comment:
Does this need to be moved?

Reply:
Changed to `get_running_loop` instead of `get_event_loop`, so just moved closer to first occurrence.

Although I'm fine reverting it if you wish.  There are a ton of occurrences of `get_event_loop` that should probably be changed.

Reply:
I think its a good change to switch to `get_running_loop`, but I'd leave it above and swap out `loop = asyncio.get_event_loop()` for `asyncio.get_running_loop()` to keep it easier to backport